src/main/resources/LabelsBundle_cy.properties

```diff
@@ -341,6 +342,7 @@ terms-conditions=Telerau ac amodau
 cookies=Briwsion a phreifatrwydd
 careers=Gyrfaoedd
 contact-us=Cysylltu â ni
+news=Newyddion
 connect=Cysylltu â ni
```

Comment:
Same question here. I see a couple of the other translations are listed twice as well. Is this a babbage thing?